pom.xml

```diff
@@ -82,6 +76,11 @@
             <artifactId>springdoc-openapi-ui</artifactId>
             <version>${org.springdoc.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Please, remove this. You don't need this dependency - we have JUnit5 from `spring-boot-starter-test`